README.md

```diff
@@ -64,6 +64,11 @@ cloudflared tunnel --hostname {your-argo-tunnel} --url http://localhost:3000/
 <script src="https://{your-argo-tunnel}/static/js/bundle.js" type="text/javascript"></script>
 ```
 
+### Insert div into subscription management page
+- Navigate to the page for managing subscriptions
+- Within the code editor, insert: 
+    
+    <div id="subscriptions-customer-portal-root"></div>
```

Comment:
For Big Commerce for instance we open the code editor and then have to click a specific button to add HTML code. Might be worth adding that extra note so people don't throw in that snippet to the main editor (it won't do anything).